sklearn/tests/test_multioutput.py

```diff
@@ -615,6 +615,7 @@ def fit(self, X, y, sample_weight=None, **fit_params):
         return super().fit(X, y, sample_weight)
 
 
+@pytest.mark.filterwarnings("ignore:`n_features_in_` is deprecated")
```

Comment:
This one is worrying me. Here, it shows that a user will see the warning for some meta-estimators using the dummy estimator. Indeed, for the meta-estimators setting `n_features_in_` depending on the inner estimator, we will check `n_features_in_` and it will raise the warning with the dummy estimators.

It might be less of a big deal because it is only a dummy estimator and maybe not that much people are using it in a meta-estimator.

Reply:
If it gets annoying for users, we can define our own `hasattr` that ignores future warnings emitted by sklearn.